src/schemas/v2/combined.json

```diff
@@ -8102,30 +8269,40 @@
             },
             "typeOfConstruction": {
               "baspiRef": "A1.8",
+              "ntsRef": "A1.2",
               "title": "Type of Construction",
               "type": "object",
               "baspiRequired": [
                 "isStandardForm",
-                "buildingSafetyRemediation",
+                "buildingSafety",
                 "sprayFoamInsulation"
               ],
+              "ntsRequired": [
+                "isStandardForm",
+                "buildingSafety",
+                "accessibilityAndAdaptation"
+              ],
               "properties": {
                 "isStandardForm": {
                   "baspiRef": "A1.8.1",
+                  "ntsRef": "A1.2.1",
                   "rdsRef": "Buildings,BuildingSections/ConstructionType,ConstructionMaterials",
                   "piqRef": "A5.5",
                   "title": "Is the property built in a standard form of construction? E.g. brick and/or block with timber-supported roof comprising a tile or slate cover?",
+                  "description": "Examples of non-standard features which might affect a buyer are are thatched roofs, prefabricated buildings or timber-framed windows",
```

Comment:
Unless there is a reason for the example being in the title propose the following description:
```
"title": "Is the property built in a standard form of construction?",
"description": "Standard construction is brick and/or block with timber-supported roof comprising a tile or slate cover. Examples of non-standard features which might affect a buyer are are thatched roofs, prefabricated buildings or timber-framed windows",
```